pyproject.toml

```diff
@@ -156,6 +156,23 @@ exclude = '''
 profile = "black"
 multi_line_output = 3
 
+[tool.ruff]
+line-length = 115
+select = ["E"]
+ignore = ["E203"]
```

Comment:
Ruff is complaining about this ignore code

Reply:
Addressed in 75156c798341938de94e1e2957120b9603d116a8